examples/electron/webpack.config.js

```diff
@@ -46,6 +47,10 @@ module.exports = {
                 loader: 'source-map-loader'
             },
             {
+                test: /node_modules.+xterm.+\.map$/,
+                loader: 'ignore-loader'
+            },
```

Comment:
ditto

Reply:
Done, removed and fill with another issue #538